specification/managedccf/data-plane/Microsoft.mCCF/preview/2023-03-03-preview/openapi.json

```diff
@@ -0,0 +1,1912 @@
+            }
+          }
+        }
+      }
+    },
+    "/gov/kv/proposals_info": {
+      "get": {
+        "operationId": "GetProposalsInfo",
+        "parameters": [],
+        "responses": {
+          "200": {
+            "description": "The request has succeeded.",
+            "schema": {
+              "$ref": "#/definitions/StringToProposalInfo"
+            }
+          }
+        }
+      }
+    },
+    "/gov/kv/service/acme_certificates": {
+      "get": {
+        "operationId": "GetServiceAcmeCertificates",
+        "parameters": [],
+        "responses": {
+          "200": {
+            "description": "The request has succeeded.",
+            "schema": {
+              "$ref": "#/definitions/StringToPem"
+            }
+          }
+        }
+      }
+    },
+    "/gov/kv/service/config": {
+      "get": {
+        "operationId": "GetServiceConfig",
+        "parameters": [],
+        "responses": {
+          "200": {
+            "description": "The request has succeeded.",
+            "schema": {
+              "$ref": "#/definitions/ServiceConfiguration"
+            }
+          }
+        }
+      }
+    },
+    "/gov/kv/service/info": {
+      "get": {
+        "operationId": "GetServiceInfo",
+        "parameters": [],
+        "responses": {
+          "200": {
+            "description": "The request has succeeded.",
+            "schema": {
+              "$ref": "#/definitions/ServiceInfo"
+            }
+          }
+        }
+      }
+    },
+    "/gov/kv/service/previous_service_identity": {
+      "get": {
+        "operationId": "GetServicePreviousServiceIdentity",
+        "parameters": [],
+        "responses": {
+          "200": {
+            "description": "The request has succeeded.",
+            "schema": {
+              "$ref": "#/definitions/Pem"
+            }
+          }
+        }
+      }
+    },
+    "/gov/kv/tls/ca_cert_bundles": {
+      "get": {
+        "operationId": "GetTlsCaCertBundles",
+        "parameters": [],
+        "responses": {
+          "200": {
+            "description": "The request has succeeded.",
+            "schema": {
+              "type": "string"
+            }
+          }
+        }
+      }
+    },
+    "/gov/kv/users/certs": {
+      "get": {
+        "operationId": "GetUsersCerts",
+        "parameters": [],
+        "responses": {
+          "200": {
+            "description": "The request has succeeded.",
+            "schema": {
+              "$ref": "#/definitions/UserIdToPem"
+            }
+          }
+        }
+      }
+    },
+    "/gov/kv/users/info": {
+      "get": {
+        "operationId": "GetUsersInfo",
+        "parameters": [],
+        "responses": {
+          "200": {
+            "description": "The request has succeeded.",
+            "schema": {
+              "$ref": "#/definitions/UserDetails"
+            }
+          }
+        }
+      }
+    },
+    "/gov/proposals": {
```

Comment:
This was intentional, since `/gov/proposals` has a nicer output format. And `/gov/kv/proposals` is a by-product of exposing all tables under `/gov/kv`

Reply:
`/gov/proposals` is the interface users should call, and only returns `OPEN` proposals. `/gov/kv/proposals` just exposes the raw table, which always also contains the last closed proposal (so that the outcome is written to the ledger).